### PR TITLE
Add email with code submitter credentials

### PR DESCRIPTION
**Pull request by RealOrangeOne**
Send an email to teams with their code submitter credentials.

As of opening this PR, teams who have already registered for friendlies have already been sent their password over Discord.

### CODE REVIEW

File: SR2021/2021-01-08-code-submitter-credentials.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+to: SR2021 teams
+subject: Student Robotics Code Submitter Credentials
+---
+
+Hi,
+
+Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.
+
+The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note what anyone in your team can submit code to the submitter, but only the latest upload will be used.
```
Comment (WillB97) [inline SR2021/2021-01-08-code-submitter-credentials.md:10]:
```suggestion
The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
```

File: SR2021/2021-01-08-code-submitter-credentials.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+subject: Student Robotics Code Submitter Credentials
+---
+
+Hi,
+
+Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.
+
+The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
+
+| TLA | Password |
```
Comment (WillB97) [inline SR2021/2021-01-08-code-submitter-credentials.md:12]: It's probably worth mentioning somewhere that their TLA is their username.

Comment (RealOrangeOne) [reply SR2021/2021-01-08-code-submitter-credentials.md:12]: aha, good catch!

File: SR2021/2021-01-08-code-submitter-credentials.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+subject: Student Robotics Code Submitter Credentials
+---
+
+Hi,
+
+Here are your credentials for using the [code submitter](https://studentrobotics.org/code-submitter/), the tool we're using to allow you to submit your code to us for running in a competition event.
+
+The code should be in a zip file, and the entry point for your robot program named `robot.py`. Note that anyone in your team can submit code to the submitter, but only the latest upload will be used.
+
+When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and the password is:
```
Comment (sedders123) [inline SR2021/2021-01-08-code-submitter-credentials.md:12]: I don't think anyone would try and copy paste a table as a password but maybe better to be explicit?
```suggestion
When visiting the code submitter, you'll be prompted for a username and password. The username will be your TLA, and your password can be found in the table below:
```

File: SR2021/2021-01-08-code-submitter-credentials.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+| TLA | Password |
+| --- | -------- |
+| ABC | Test     |
+
+Note that both usernames and passwords are case sensitive.
+
+## Friendlies
+
+If you're interested in competing in the friendlies this weekend you must let us know beforehand via email or discord, so we know to schedule you some matches. Teams who submit code but haven't signed up won't be entered. These matches are entirely optional, and will not contribute to the league. We'll use a YouTube [livestream](https://youtu.be/iwa-7R6sR8M) to broadcast the matches and as usual we'll provide your logs afterwards. Unlike our other events, these matches will be incredibly relaxed and informal. We encourage anyone to join in the conversation in Discord or in the chat.
```
Comment (sedders123) [inline SR2021/2021-01-08-code-submitter-credentials.md:22]: I don't think our ordinary events are that formal (though they undoubtedly have more structure). Feel free to ignore if you disagree.
```suggestion
If you're interested in competing in the friendlies this weekend you must let us know beforehand via email or discord, so we know to schedule you some matches. Teams who submit code but haven't signed up won't be entered. These matches are entirely optional, and will not contribute to the league. We'll use a YouTube [livestream](https://youtu.be/iwa-7R6sR8M) to broadcast the matches and as usual we'll provide your logs afterwards. These matches will be incredibly relaxed and informal and, as ever, we encourage anyone to join in the conversation in Discord or in the chat.
```

**Review by sedders123 — APPROVED** [review]
Looks good, just one potential tweak

File: SR2021/2021-01-08-code-submitter-credentials.md (hunk 4 above)
Comment (trickeydan) [reply SR2021/2021-01-08-code-submitter-credentials.md:22]: :+1:

**Review by sedders123 — APPROVED** [review]

**Review by PeterJCLaw — APPROVED** [review]

**Pull request merged**